### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 9

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#297] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-10-28 110538](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/2dd0f6ff-b437-4828-8016-3bc48f39b2bf)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/cc88e792-9b4d-468b-ab6a-eea90be247b1)
Soluzione alternativa diversa:
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/4934763c-4ad0-49e9-a79a-748ae2a00073)
